tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/optimizer/OptimizerCore.scala

```diff
@@ -2251,50 +2245,61 @@ private[optimizer] abstract class OptimizerCore(
                  * will not go into infinite recursion.
                  */
                 implicit val scope = Scope.Empty
-                inline(Nil, Some(PreTransTree(x)), Nil, target, isStat = false,
+                inline(Nil, Some(x), Nil, target, isStat = false,
                     usePreTransform = false)(finishTransform(isStat = false))
-              }
+              })
```

Comment:
This makes me think: Why do the fold operations not take continuations like all methods on pretransforms?